miio/gateway.py

```diff
@@ -202,267 +230,289 @@ def set_developer_key(self, key):
+        """
+        Return time in seconds the alarm stays 'oning'
+        before transitioning to 'on'
+        """
         # Response: 5, 15, 30, 60
-        return self._device.send("get_arm_wait_time").pop()
+        return self._gateway.send("get_arm_wait_time").pop()
 
     @command(click.argument("seconds"))
     def set_arming_time(self, seconds):
         """Set time the alarm stays at 'oning' before transitioning to 'on'"""
-        return self._device.send("set_arm_wait_time", [seconds])
+        return self._gateway.send("set_arm_wait_time", [seconds])
 
     @command()
     def triggering_time(self) -> int:
         """Return the time in seconds the alarm is going off when triggered"""
         # Response: 30, 60, etc.
-        return self._device.send("get_device_prop", ["lumi.0", "alarm_time_len"]).pop()
+        return self._gateway.get_prop("alarm_time_len").pop()
 
     @command(click.argument("seconds"))
     def set_triggering_time(self, seconds):
         """Set the time in seconds the alarm is going off when triggered"""
-        return self._device.send(
-            "set_device_prop", {"sid": "lumi.0", "alarm_time_len": seconds}
-        )
+        return self._gateway.set_prop("alarm_time_len", seconds)
 
     @command()
     def triggering_light(self) -> int:
-        """Return the time the gateway light blinks when the alarm is triggerd"""
+        """
+        Return the time the gateway light blinks
+        when the alarm is triggerd
+        """
         # Response: 0=do not blink, 1=always blink, x>1=blink for x seconds
-        return self._device.send("get_device_prop", ["lumi.0", "en_alarm_light"]).pop()
+        return self._gateway.get_prop("en_alarm_light").pop()
 
     @command(click.argument("seconds"))
     def set_triggering_light(self, seconds):
         """Set the time the gateway light blinks when the alarm is triggerd"""
         # values: 0=do not blink, 1=always blink, x>1=blink for x seconds
-        return self._device.send(
-            "set_device_prop", {"sid": "lumi.0", "en_alarm_light": seconds}
-        )
+        return self._gateway.set_prop("en_alarm_light", seconds)
 
     @command()
     def triggering_volume(self) -> int:
         """Return the volume level at which alarms go off [0-100]"""
-        return self._device.send("get_alarming_volume").pop()
+        return self._gateway.send("get_alarming_volume").pop()
 
     @command(click.argument("volume"))
     def set_triggering_volume(self, volume):
         """Set the volume level at which alarms go off [0-100]"""
-        return self._device.send("set_alarming_volume", [volume])
+        return self._gateway.send("set_alarming_volume", [volume])
 
     @command()
     def last_status_change_time(self):
-        """Return the last time the alarm changed status, type datetime.datetime"""
-        return datetime.fromtimestamp(self._device.send("get_arming_time").pop())
+        """
```

Comment:
Could you make this (and other similar cases, if any) one-liners by removing datetime.timetime and defining it as return value type hint.

Reply:
done